CTFd/api/v1/config.py

```diff
@@ -109,8 +109,12 @@ def post(self):
     )
     def patch(self):
         req = request.get_json()
+        schema = ConfigSchema()
 
         for key, value in req.items():
+            response = schema.load({"key": key, "value": value})
+            if response.errors:
+                return {"success": False, "errors": response.errors, "key": key}, 400
```

Comment:
I think adding the key is the wrong idea here because it breaks the expectation of success, data, and errors that all the API endpoints are using. We can just add the key into the error.